README.md

```diff
@@ -1 +1,157 @@
-# Metrics-webapp
\ No newline at end of file
+# Metrics webapp
+<!-- TABLE OF CONTENTS -->
+
+<details>
+  <summary>📗 Table of Contents</summary>
+  <ul>
+    <li><a href="#-Metrics-webapp">📖 Metrics webapp</a></li>
+    <li><a href="#-built-with-">🛠 Built With</a></li>
+    <li><a href="#-live-demo-">🚀 Live Demo </a></li>
+    <li><a href="#-getting-started-">💻 Getting Started</a></li>
+    <li><a href="#-author-">👥 Author </a></li>
+    <li><a href="#-future-features-">🔭 Future Features</a></li>
+    <li><a href="#-contributing-">🤝 Contributing</a></li>
+    <li><a href="#️-show-your-support-">⭐️ Show your support </a></li>
+    <li><a href="#-acknowledgments-">🙏 Acknowledgments </a></li>
+    <li><a href="#-license-">📝 License</a></li>
+  </ul>
+</details>
+
+<br>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Metrics webapp <a name="about-project"></a>
```

Comment:
- [ ] [REQUIRED] It is a requirement that you record a video presenting the features of the project. The maximum length of the video is 5 minutes. The video should contain a clear and objective description of the project in professional English. The video contains a demo of the project, highlighting all the project features. The video contains a highlight about a piece of the code that you are proud of making.